### DISCUSSION TITLE
Linux wouldn't compile

### DISCUSSION THREAD

**Discussion by rahard** [asb2m10/dexed#281] [Q&A]
I am trying to build Dexed from source code (from github). After following the instruction, I am stuck at this point.
```

./scripts/build-lin.sh 
Makefile:153: *** target pattern contains no '%'.  Stop.
```

Any pointer(s)

**Comment by asb2m10** [comment]
Please see #284 for this... otherwise this should not be an issue if it is built with cmake.

> The error comes from line 153 of the generated Makefile you get from projucer. That line looks like this:
> 
> VST3 : $(JUCE_OUTDIR)/$(JUCE_TARGET_VST3)
> tldr:
> You should install C++ compiler. I use debian based distro and install C++ like this:
> 
> sudo apt install g++
> Explanation
> Varialble JUCE_TARGET_VST3 to be populated involves another variable CXX. And variable CXX populated when C++ path exists.
> Otherwise JUCE_TARGET_VST3 remains empty and build script throws target pattern contains no '%'.
